src/sbt-test/avrohugger/GenericSerializationTests/src/test/scala/test/standard/StandardPrimitivesSpec.scala

```diff
@@ -144,5 +145,16 @@ class StandardPrimitivesSpec extends Specification {
       StandardTestUtil.verifyWriteAndRead(records)
     }
   }
+  
+  // Aren't backticks supposed to solve this?
```

Comment:
This is fixed in julianpeeters/avrohugger#125, with a very strange caveat that I can't seem to figure out. When compiling, dependencies in the `Test` scope fail to be compiled in as well. I don't see how my changes cause that, but it's also happening on `master`, so it might just be environmental.

Reply:
That sounds like a bug introduced in sbt-avrohugger 2.0.0-RC16 and fixed in 2.0.0-RC17